pkg/kubelet/metrics/metrics.go

```diff
@@ -28,13 +28,6 @@ import (
 const kubeletSubsystem = "kubelet"
 
 var (
-	ImagePullLatency = prometheus.NewSummary(
```

Comment:
I am okay with removing this for now if it gets in the way. We can discuss how to implement this later, as long as we don't depend on the metric being here for v1.0.